translate/py_main.py

```diff
@@ -13,11 +13,11 @@
 takes pyton file and converts it to a node
 node is then dealt with by node_to_gast 
 """
-def py_to_gast(python_input_filename):
-    input_ast =  astor.code_to_ast.parse_file(python_input_filename)
+def py_to_gast(python_input):
+    input_ast = ast.parse(python_input)
     print(astor.dump_tree(input_ast))
     return py_router.node_to_gast(input_ast)
 
-fileName = "py_sample_code.py"
-gast = py_to_gast(fileName)
+file = "print('hello world')"
```

Comment:
Can we rename this to `src_code_input` (or something similar) now that this isn't a file anymore?

Reply:
`python_input` like in `py_to_gast` works

Reply:
got it